peewee field with choices

Peewee field with a `choices` argument is not passed to GraphQL.

For example we can modify [model](https://github.com/insolite/graphene-peewee-async/blob/master/tests/common/models.py#L24) from tests:
```python
GENRES = (
    (0, 'Science fiction'),
    (1, 'Guide'),
    (2, 'Horror'),
)

class Book(BaseModel):
    name = CharField()
    year = IntegerField()
    author = ForeignKeyField(Author)
    genre = IntegerField(choices=GENRES)
```
and modify [test](https://github.com/insolite/graphene-peewee-async/blob/master/tests/test_api/test_query.py#L10):
```python
        book = self.loop.run_until_complete(
            self.manager.create(Book, name='bar', year=2000, author=author, genre=1)  # genre was added
        )

        result = self.loop.run_until_complete(self.query('''
            query {
                book (id: ''' + str(book.id) + ''') {
                    id
                    name
                    year
                    genre  # this
                    author {
                        id
                        name
                        rating
                    }
                }
            }
        '''))
```

unittest:
`AssertionError: [GraphQLError('Cannot query field "genre" on type "Book".',)] is not None`

I tried to figure point of this error and found what [function](https://github.com/graphql-python/graphene/blob/f1624af08aec4344b365094951aefc9979bced07/graphene/types/utils.py#L12) from graphene return None.

So. What's wrong with my code?

--- a/graphene_peewee_async/converter.py
+++ b/graphene_peewee_async/converter.py
@@ -28,12 +28,13 @@ def get_foreign_key_id_field(field):
 
 
 def convert_peewee_field_with_choices(field, registry=None):
-    choices = getattr(field, 'choices', None)
-    if choices:
-        meta = field.model_class._meta
-        name = '{}_{}'.format(meta.name, field.name)
-        graphql_choices = list(convert_choices(choices))
-        return Enum(name.upper(), graphql_choices, description=field.help_text)
+    # TODO: Fix choices in a good way. See https://github.com/insolite/graphene-peewee-async/issues/5
+    # choices = getattr(field, 'choices', None)
+    # if choices:
+    #     meta = field.model_class._meta
+    #     name = '{}_{}'.format(meta.name, field.name)
+    #     graphql_choices = list(convert_choices(choices))
+    #     return Enum(name.upper(), graphql_choices, description=field.help_text)()
     return convert_peewee_field(field, registry)
 
 

